bouncer/blacklist/migrations/0002_auto_20180725_1038_squashed_0004_emailhostentry_lower_case_entry_value.py

```diff
@@ -1,10 +1,16 @@
-# Generated by Django 2.0.7 on 2018-07-19 11:33
+# Generated by Django 2.0.7 on 2018-07-25 11:34
```

Comment:
Since these migrations are all part of the same PR, it would be best to just include one. In order to do this:

* Remove all migration files in this PR
* Run `makemigrations` again
* Commit the changes

Locally, you will need to run the new migrations with `--fake`, since the field will have already been created.